automaton.py

```diff
@@ -1,37 +1,140 @@
+import enum
+
 class Transition():
 
-    def __init__(self, dest: str, target: str, symbol: str):
+    def __init__(self, dest: str, target: list=[], symbol: list=[]):
         self.dest = dest
         self.target = target
         self.symbol = symbol
 
     def render(self):
         # TODO: Return a string in the format s0:a>s1
         pass
+        
+class operationID(enum.Enum):
+    INVALID = 0
+    STATES = 1
+    INITIAL = 2
+    ACCEPT = 3
+    ALPHABET = 4
+    TRANSITIONS = 5
+    
+    
 
 class Automaton():
 
     def __init__(self, states: list = [], initial: str = "", accept: list = [], alphabet: list = [], transitions: list=[], from_file: str = ""):
-        self.states = states
-        self.initial = initial
-        self.accept = accept
-        self.alphabet = alphabet
-        self.transitions = transitions
```

Comment:
Removing this type of initialization means that parameters are no longer required